baggage-processor/build.gradle.kts

```diff
@@ -0,0 +1,16 @@
+plugins {
+  id("otel.java-conventions")
+
+  id("otel.publish-conventions")
+}
+
+description = "OpenTelemetry Baggage Span Processor"
+otelJava.moduleName.set("io.opentelemetry.contrib.baggage.processor")
+
+dependencies {
+  api("io.opentelemetry:opentelemetry-api")
+  api("io.opentelemetry:opentelemetry-sdk")
+  api("io.opentelemetry:opentelemetry-sdk-extension-autoconfigure-spi")
```

Comment:
Is this used?

```suggestion
```

Reply:
Not currently, it can be removed. I would like to add support for it so the java agent could also utilise this component too, but we can do that as a follow up PR.